templates/certification/index.html

```diff
@@ -130,9 +120,10 @@ <h2 class="p-card__title p-heading--3">Ubuntu Desktop certified hardware</h2>
         loading="lazy"
```

Comment:
Maybe I don't understand this functionality well enough, but why are you using `lazy` in some and `auto` in others?

Reply:
I just thought as the top two could be visible on page load for some screens better not lazy load them